README.md

```diff
@@ -60,6 +60,18 @@ _Protip_: `element update -p`
 
 * * *
 
+### `element release`
+
+When you are ready to push your block live you can use this command.
+
+### `element release --note "release note for the block`
```

Comment:
How do you feel about making the release note mandatory?

element release
> Release note: Enter release note

Reply:
I mean I guess they could just write garbled anything and release. What if on the UI we just had some good default release note text?

Reply:
Sounds good.